--- hashlib.py  2015-12-25 15:46:59.948170139 +0000
+++ /omd/versions/1.20/lib/python/hashlib.py    2015-12-25 15:24:39.268207021 +0000
@@ -115,6 +115,8 @@
             f()
             # Use the C function directly (very fast)
             exec funcName + ' = f'
+        except TypeError:
+            pass
         except ValueError:
             try:
                 # Use the builtin implementation directly (fast)

