--- cv_install_mail	2015-02-25 12:57:27.554135836 +0100
+++ /root/install_scripts/monitoring/cv_install_mail	2015-02-24 14:45:24.710665930 +0100
@@ -16,7 +16,7 @@
 postconf -e "myorigin = \$mydomain"
 postconf -e "inet_interfaces = all"
 postconf -e "mydestination = \$myhostname, localhost.\$mydomain, localhost"
-postconf -e "mynetworks = 192.168.0.0/24, 127.0.0.0/8"
+postconf -e "mynetworks = 10.141.0.0/16, 127.0.0.0/8"
 postconf -e "relay_domains = "
 postconf -e "home_mailbox = Maildir/"
 
